### PR TITLE
Add skill forge lab, /mobius-profile skill, and ideas backlog

### PR DESCRIPTION
**Pull request by AaronGoldsmith**
## Summary

- **`labs/2026-03-14-skill-forge.md`** — First two-phase competition: 6 agents designed Claude Code skills, then 6 testers functionally validated them against the live DB. Full results, scores, and findings documented.
- **`/mobius-profile`** — Competition winner. Single-script skill that shows agent deep-dives with match history, win/loss analysis, and challenger recommendations. Tested and working.
- **`ideas.md`** — Living backlog of concepts that emerged from the competition (forge skill, match replay, agent factory, self-audit).

## Key Finding

The **generate → test** two-phase pattern was more valuable than any individual skill. Design scores and functional scores diverged significantly — one skill scored 2nd in design but last in functional testing (shipped with crashing bugs). This pattern should become repeatable.

Relates to #5 (mobius improve self-diagnosing loop)

## Test plan

- [x] `/mobius-profile` tested against live DB with 40 agents
- [x] Edge cases verified (0 matches, nonexistent agents)
- [x] Lab entry reviewed for accuracy against actual competition results

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: .claude/skills/mobius-profile/scripts/show_profile.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+"""
+
+import json
+import sys
+from collections import Counter
+
+sys.path.insert(0, "src")
+
+from mobius.config import get_config
+from mobius.db import init_db, row_to_dict
```
[the remark this answers is not in the document]
Comment (AaronGoldsmith) [reply .claude/skills/mobius-profile/scripts/show_profile.py]: Fixed in 16d0dd2 — removed unused `json` and `row_to_dict` imports.

File: .claude/skills/mobius-profile/scripts/show_profile.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            opponent_names = []
+            for opp_id in opponents:
+                opp = registry.get_agent(opp_id)
+                opp_name = opp.name if opp else opp_id[:8]
+                opponent_names.append(opp_name)
+
+                if is_win:
+                    opponent_wins[opp_name] += 1
+                else:
+                    opponent_losses[opp_name] += 1
```
[the remark this answers is not in the document]
Comment (AaronGoldsmith) [reply .claude/skills/mobius-profile/scripts/show_profile.py]: Fixed in 16d0dd2 — losses are now only counted against the actual match winner (`opp_id == match.winner_id`), not all opponents.

[the remark this answers is not in the document]
Comment (AaronGoldsmith) [reply .claude/skills/mobius-profile/scripts/show_profile.py]: Fixed in 16d0dd2 — counters are now keyed by `opp.slug` with a separate `slug_to_name` map for display.

File: labs/2026-03-14-skill-forge.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+**Task:** Design a Claude Code skill for Mobius that saves developer time or improves the system. Output a complete SKILL.md with frontmatter and helper scripts.
+
+**6 agents, 2 tracks:**
+
+### Scouts (web research → adapt existing patterns)
+| Slug | Philosophy | Tools |
+|------|-----------|-------|
+| `skill-scout-trending` | Find high-traction GitHub skills, adapt for agent swarms | WebSearch, WebFetch, Read, Grep, Glob, Bash |
+| `skill-scout-ecosystem` | Research Claude Code ecosystem, find underutilized capabilities | WebSearch, WebFetch, Read, Grep, Glob, Bash |
+| `skill-scout-devtools` | Find dev tool integrations, wrap multi-step workflows | WebSearch, WebFetch, Read, Grep, Glob, Bash |
```
[the remark this answers is not in the document]
Comment (AaronGoldsmith) [reply labs/2026-03-14-skill-forge.md]: Inspected the file — the tables use standard single-pipe markdown. This appears to be a false positive; no changes needed.

File: .claude/skills/mobius-profile/scripts/show_profile.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (AaronGoldsmith) [reply .claude/skills/mobius-profile/scripts/show_profile.py]: Fixed in 16d0dd2 — same fix as above. Losses now only count against `match.winner_id`, and voided matches are skipped entirely.
